guides/common/modules/proc_updating-operating-system.adoc

```diff
@@ -0,0 +1,36 @@
+[id="updating-operating-system_{context}"]
+= Updating the operating system
+
+{Team} recommends performing operating system updates regularly, because the minor releases patch security vulnerabilities and minor issues discovered after code is released.
+
+.Prerequisites
+* Back up your {ProjectServer}.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
+
+include::snip_warning-maintain-config-noop.adoc[]
+
+.Procedure
+. Use the health check option to determine if the system is ready for update.
+On first use of this command, `{foreman-maintain}` prompts you to enter the hammer admin user credentials and saves them in the `/etc/foreman-maintain/foreman-maintain-hammer.yml` file.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} update check
+----
++
+Review the results and address any highlighted error conditions before performing the update.
+. Because of the lengthy update time, use a utility such as `tmux` to suspend and reattach a communication session.
+You can then check the update progress without staying connected to the command shell continuously.
++
+If you lose connection to the command shell where the update command is running, you can see the logged messages in the `{installer-log-file}` file to check if the process completed successfully.
+. Perform the update:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} update run
```

Comment:
What happens if there are not only OS but also Satellite updates outstanding?
Will this ensure Satellite is (not?) updated? (I hope it won't, as we often ship fixes that are required for "latest BaseOS", but the wording in this chapter implies otherwise).

Reply:
It will include them - I added a note.

Reply:
It's not really - but users don't understand that it can be used for either, and I figured a top level section dedicated to it increases its visibility and ability for users to understand it.